### ISSUE TITLE
Pagination should be added to URL as query string.

### ISSUE THREAD

**Issue by jojo-tutor** [odota/web#1446]
In this link: https://www.opendota.com/teams/2163/matches, on the **"Matches"** and **"Heroes"** Tab, there are pagination, if you should go to other pages, you must add the current page to the URL, so that whenever the user will refresh the browser/share the URL, the current page will retain accordingly. This is a good practice. And also, in pagination, it's good to have an option like: **results per page**.

![opendota](https://user-images.githubusercontent.com/18729213/36236252-155ebbae-1230-11e8-9bde-ec67501dc0f7.jpg)

**Comment by howardchung** [comment]
Yes, I think we can definitely add a page parameter that's bound to the PaginatedTable component and saves the state in the URL

**Comment by gar-stauffer** [comment]
Hey, I'll pick this up. I still need to set up the OD environment; will work on this issue over course of week

**Comment by coreymaher** [comment]
The problem with using the page number in the url when sorting descending is it gets out of date very quickly. You may be trying to share a link for a match on page 2, but tomorrow the match has been pushed down to page 3.

Is this a common use case that you want to support? If so, it's probably better to store something that can generate the same page list each time, for example the topmost match id.
